src/book-search-result.js

```diff
@@ -0,0 +1,38 @@
+import { html, LitElement } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+
+export class BookSearchResult extends LitElement {
+  static get properties() {
+    return {
+      match: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.matchRegex = new RegExp('{{{(.+?)}}}', 'g');
+  }
+
+  createRenderRoot() {
+    return this;
+  }
+
+  highlightedHit(hit) {
+    return html`
+      <p>${unsafeHTML(hit.replace(this.matchRegex, '<mark>$1</mark>'))}</p>
+    `;
+  }
+
+  render() {
+    const { match } = this;
+    const coverImage = html`<img src="${match.cover}" />`;
+    return html`
+      <li>
```

Comment:
I might move the `<li>` up to the parent since it is putting the container in place with its `<ul>`.

Reply:
Moved it over.